django/contrib/gis/db/models/fields.py

```diff
@@ -214,12 +214,12 @@ def get_prep_value(self, value):
                 try:
                     obj = GDALRaster(obj)
                 except GDALException:
-                    raise ValueError("Couldn't create spatial object from lookup value.")
+                    raise ValueError("Couldn't create spatial object from lookup value \"%s\"." % obj)
```

Comment:
We typically use single quotes for either the inner or outer string to avoid backslashes.